core/domain/value_generators_domain.py

```diff
@@ -45,15 +45,26 @@ def id(self):
 
     @classmethod
     def get_html_template(cls):
+        """Returns the HTML template for the class.
+
+        Returns:
+            The HTML template corresponding to the class,
+        """
         return utils.get_file_contents(os.path.join(
             os.getcwd(), feconf.VALUE_GENERATORS_DIR, 'templates',
             '%s.html' % cls.__name__))
 
     @classmethod
     def get_js_template(cls):
-        # NB: These generators should use only Angular templating. The
-        # variables they have access to are generatorId, initArgs,
-        # customizationArgs and objType.
+        """Returns the JavaScript template for the class.
+
+        Returns:
+            The JS template corresponding to the class,
+
+        NB: These generators should use only Angular templating. The
```

Comment:
Can we convert this to a comment above the "return" statement? It's not really part of the docstring.